scripts/ci/deploy.sh

```
@@ -129,8 +130,40 @@ health_check() {
+  local template="$PROJECT_DIR/infra/nginx/rateukma.conf.template"
+  local dest="/etc/nginx/sites-available/${SERVER_NAME}"
+  local backup="/tmp/nginx-backup-${SERVER_NAME}.conf"
+
+  if [ ! -f "$template" ]; then
+    echo "Nginx template missing at $template" >&2
+    return 1
+  fi
+
+  [ -f "$dest" ] && sudo cp "$dest" "$backup"
+
+  envsubst '${SERVER_NAME}' < "$template" | sudo tee "$dest" > /dev/null
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)